src/platform/packages/shared/kbn-unified-tabs/src/utils/calculate_responsive_tabs.test.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { calculateResponsiveTabs } from './calculate_responsive_tabs';
+import { getNewTabPropsForIndex } from '../hooks/use_new_tab_props';
+
+const items = Array.from({ length: 5 }).map((_, i) => getNewTabPropsForIndex(i));
+
+describe('calculateResponsiveTabs', () => {
+  it('renders a single tab without limitation', () => {
+    const tabsSizeConfig = calculateResponsiveTabs({ items: [items[0]], containerWidth: 1500 });
+
+    expect(tabsSizeConfig).toEqual({
+      isScrollable: false,
+      regularTabMaxWidth: 280,
+      regularTabMinWidth: 96,
```

Comment:
What do you think about extracting it to const? Most values repeat quite often within the test suite

Reply:
Or importing it from `calculate_responsive_tabs.ts`

Reply:
Addressed in https://github.com/elastic/kibana/pull/213739/commits/f25ee3cc3674f70058707c6939469e47802e0a5b
Thanks!